src/custom/simplifions/components/SimplifionsSolutionDescription.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <div class="">
+    <p class="fr-text--small">
```

Comment:
Avis perso : un peu surpris du choix de mise en page : 
- Description (courte) avant titre
- Description courte en petit vs longue en gros.

Reply:
C'était le choix de l'UI/UX Designer, je la laisserai itérer sur ça.